### PR TITLE
repo: add make dist, make distcheck support to flux-accounting

### PR DESCRIPTION
**Pull request by cmoussa1**
One of the things discussed in the comments in #151 was in order to build an RPM for future versions of flux-accounting, adding support for `make dist` and `make distcheck` to build a `.tar.gz` would make building on systems easier. Currently, the repository doesn't support this, for a majority of reasons because of some of the repo's `Makefile`s and source files. Since I am targeting `v0.10.0` as the first release to build and test on a system, it would be good to have this support of building a `.tar.gz` working before we release `v0.10.0`.

This PR makes several changes to the flux-accounting repository in order to add support for running `make dist` and `make distcheck`. It changes some of the `Makefile`s, changes some of the prefixes in the sharness tests, and changes some of the `import` functionality in the front-end Python script. 

This PR is left as a [WIP] for now until I can clean up/re-order (if needed) the commits on this branch a little bit, but eventually I think this PR should go in before #151, which is the release PR.

### CODE REVIEW

**Comment by cmoussa1** [comment]
OK, I think I've got better ordering and description of the commit messages, so I'll pull this out of **[WIP]**.

File: src/cmd/Makefile.am [hunk 1]
```diff
@@ -2,13 +2,16 @@ AM_CXXFLAGS = \
     $(WARNING_CXXFLAGS) \
     $(CODE_COVERAGE_CXXFLAGS)
 
-AM_LDFLAGS = $(CODE_COVERAGE_LDFLAGS)
+AM_LDFLAGS = $(CODE_COVERAGE_LDFLAGS) $(SQLITE_LIBS)
 
-AM_CPPFLAGS = $(FLUX_CORE_CFLAGS)
+AM_CPPFLAGS = -I$(top_srcdir) $(FLUX_CORE_CFLAGS)
```
Comment (grondo) [inline src/cmd/Makefile.am:7]: Commit message:

 * typo: `Soltuion` :arrow_right: `Solution`
 * Suggestion: use C++ "source files" or "sources" not "scripts"

**Review by grondo — APPROVED** [review]
LGTM, and I verified `make distcheck` and a vpath build worked for me.

Just one nit for one of the commit messages.

You may want to add `DISTCHECK=t` to one of the builders to ensure `make distcheck` won't break in the future, and to produce a tarball for auto-deploy. However, that can be done in a future PR.

File: src/cmd/Makefile.am (hunk 1 above)
Comment (cmoussa1) [reply src/cmd/Makefile.am:7]: Oops 🤦‍♂️ thanks for catching this. I just pushed up a fix to address the typos and wrong word choice for the commit message.

**Comment by cmoussa1** [comment]
Thanks @grondo for reviewing this! I'll open a couple of issues on adding `DISTCHECK=t` to one of the builders to ensure `make distcheck` won't break in the future, and to produce a tarball for auto-deploy. I'll set MWP shortly.

**Pull request merged**